src/sass/app.scss

```diff
@@ -1,26 +1,37 @@
-$fuente-web: "Gothic A1", sans-serif;
-$color-web: #747474;
-$tamano-fuente-web: 19px;
-$background-web: #F0F0F0;
-$anchopunto: 18px;
-$anchoimagen: 75px;
+$web-font: "Gothic A1", sans-serif;
+$web-color: #747474;
+$web-font-size: 19px;
+$web-background: #F0F0F0;
+$main-container-width: 750px;
+$main-container-width-1050: 650px;
```

Comment:
We should avoid using number values in variable names because that's why the variable exists in the first place (to change its value) we can rename it according to their breakpoint name, i.e. `$main-container-width-laptop: 1050;`